docs/details/develop-checks/checks-v1.md

```diff
@@ -0,0 +1,254 @@
+ - **Major**: warning  
+ - **Trivial**: notice, info, debug
+ 
+## Check fields 
+Checks can include the following fields:
+ 
+- **Version** (integer, required): defines what other properties are expected, what types are supported, what is expected from the script and what it can expect from the execution environment, etc.
+- **Name** (string, required): defines machine-readable name (ID).
+- **Summary** (string, required): defines short human-readable description.
+- **Description** (string, required): defines long human-readable description.
+- **Interval** (string/enum, optional): defines running interval. Can be one of the predefined intervals in the UI: Standard, Frequent, Rare.
+- **Type** (string/enum, required): defines the query type and the PMM Service type for which the advisor runs. Check the list of available types for version 1 checks in the table below.
+- **Script** (string, required): contains a small Starlark program that processes query results, and returns check results. It is executed on the PMM Server side.
+- **Category** (string, required): specifies a custom or a default advisor check category. For example: Performance, Security. 
+- **Query** (string, can be absent if the type defines the whole query by itself): The query is executed on the PMM Client side and contains query specific for the target DBMS.
+
+## Check types
+ 
+Expand the table below for the list of checks types that you can use to define your query type and the PMM Service type for which the check will run.
+ 
+??? note alert alert-info "Check Types table"
+ 
+    | Check type  |  Description | "query" required (must be empty if "No")   |  
+    |---|---|---|
+    | MYSQL_SHOW |Executes 'SHOW …' clause against MySQL database.  |Yes|
+    | MYSQL_SELECT    |     Executes 'SELECT …' clause against MySQL database.    |Yes|
+    | POSTGRESQL_SHOW     |    Executes 'SHOW ALL' command against PosgreSQL database.   |No|
+    | POSTGRESQL_SELECT      | Executes 'SELECT …' clause against PosgreSQL database.    |Yes|
+    | MONGODB_GETPARAMETER     | Executes db.adminCommand( { getParameter: "*" } ) against MongoDB's "admin" database. For more information, see [getParameter](https://docs.mongodb.com/manual/reference/command/getParameter/)| No|
+    | MONGODB_BUILDINFO    | Executes db.adminCommand( { buildInfo:  1 } ) against MongoDB's "admin" database.  For more information, see [buildInfo](https://docs.mongodb.com/manual/reference/command/buildInfo/) | No|
+
+## Develop version 1 checks
+To develop custom checks for PMM 2.28 and later: 
+
+1. Install the latest PMM Server and PMM Client builds following the [installation instructions](https://www.percona.com/software/pmm/quickstart#). 
+2. Run PMM Server with special environment variables:
+ 
+    - _PMM_DEBUG=1_ to enable debug output that would be useful later;
+    - _PERCONA_TEST_CHECKS_FILE=/srv/custom-checks.yml_ to use checks from the local files instead of downloading them from Percona Portal.
```

Comment:
```suggestion
    - _PERCONA_TEST_CHECKS_FILE=/srv/custom-checks.yml_ to use checks from the local files instead of downloading them from Percona Platform.
```